target_quickbooks/mapper.py

```diff
@@ -76,6 +112,9 @@ def customer_from_unified(record):
         if isinstance(addresses, str):
             addresses = eval(addresses)
 
+        if isinstance(addresses, dict):
+            addresses = [addresses]
+
```

Comment:
I think all the error handling looks good overall, but I noticed these same errors could happen in other mappings for example the address handling, it would be good to add the same handling for invoices and sales_receipt mapping as well. The same with the lookups, just to be sure it's being handled in all mappings.